doc/manual/general.rst

```diff
@@ -213,13 +213,27 @@ Sec. :ref:`Head-Tracking <head_tracking>`.
 Audio Scene Description Format (ASDF)
 -------------------------------------
 
-Besides pure audio files, SSR can also read the current development
-version of the *Audio Scene Description Format (ASDF)* [Geier2008b]_. Note,
-however,
-that so far we have only implemented descriptions of static features.
-That means in the current state it is not possible to describe
-e.g. movements of a virtual sound source.
-See https://github.com/SoundScapeRenderer/ssr/pull/155 for latest developments.
+Besides pure audio files, SSR can also read the
+*Audio Scene Description Format (ASDF)*
+Currently, two versions of the ASDF are supported.
+
+There is a legacy version of the ASDF [Geier2008b]_,
+which has been supported since the beginning,
+but which can only describe static scenes.
+Recently, support for a new version 0.4
```

Comment:
I think that it should be "Recently, support for a new version 0.4 *was* added ..."